### ISSUE TITLE
v0.01a Cleanup

### ISSUE THREAD

**Issue by prmiller91** [chapinb/anparser#7]
I was looking at the output, should the first column be the file_path variable so they know where the database is located followed by the table? Also, what do you think about making all of the header fields capitalized?

**Comment by chapinb** [comment]
Yes to both! I think in the XMl plugins, we need to re-assign the fields even though they are already bundled into output format to ensure the name is the first column and that all available fields are displayed.
